From a797d04ca69ae73477e4fc63df411689ae36c584 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Mon, 22 Sep 2014 19:41:34 -0400
Subject: [crypto] pkcs7: Add a missing static

Message-id: <1411414897-26220-2-git-send-email-vgoyal@redhat.com>
Patchwork-id: 96447
O-Subject: [RHEL7.1 PATCH 1/4] PKCS#7: Add a missing static
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>

BZ:1129779

This commit id is in linux-security tree of James Morris.

commit 15155b9a4530f8da42d6594927928e25b2b29875
Author: David Howells <dhowells@redhat.com>
Date: Tue, 16 Sep 2014 17:07:07 +0100

Add a missing static (found by checker).

Signed-off-by: David Howells <dhowells@redhat.com>
Acked-by: Vivek Goyal <vgoyal@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/asymmetric_keys/pkcs7_trust.c b/crypto/asymmetric_keys/pkcs7_trust.c
index e666eb0..fad888e 100644
--- a/crypto/asymmetric_keys/pkcs7_trust.c
+++ b/crypto/asymmetric_keys/pkcs7_trust.c
@@ -23,9 +23,9 @@
 /**
  * Check the trust on one PKCS#7 SignedInfo block.
  */
-int pkcs7_validate_trust_one(struct pkcs7_message *pkcs7,
-        struct pkcs7_signed_info *sinfo,
-        struct key *trust_keyring)
+static int pkcs7_validate_trust_one(struct pkcs7_message *pkcs7,
+        struct pkcs7_signed_info *sinfo,
+        struct key *trust_keyring)
 {
  struct public_key_signature *sig = &sinfo->sig;
  struct x509_certificate *x509, *last = NULL, *p;
-- 
1.7.1